src/util.rs

```diff
@@ -802,27 +802,36 @@ impl Run {
                 )
             }
             RunMode::Egglog => {
-                assert_eq!(self.eggcc_config.schedule, Schedule::Parallel, "Output egglog only works in parallel mode. Sequential mode does not use a single egraph");
-
                 let rvsdg = Optimizer::program_to_rvsdg(&self.prog_with_args.program)?;
                 let (dag, mut cache) = rvsdg.to_dag_encoding(true);
 
-                let schedule_steps = parallel_schedule();
-                if schedule_steps.len() != 1 {
-                    log::warn!("Parallel schedule had multiple steps! You may need to adjust the schedule to make eggcc tractable.");
-                }
+                // how many actual passes to run
+                let cutoff = self
+                    .eggcc_config
+                    .get_normalized_cutoff(parallel_schedule().len());
+                assert!(cutoff != 0);
+                let cutoff = cutoff - 1;
```

Comment:
Why are you subtracting 1 here? Shouldn't you pass it to the config as-is?

Reply:
say there are N passes and the user passes `stop-after-n-passes N` (the default value). This will run 0..(N-1) passes, and use the last schedule, which is at position `N-1`. Maybe I miss something here?

Reply:
Ah okay, I get it now